tests/test_routes.py

```diff
@@ -0,0 +1,63 @@
+from app.models.planet import Planet
+
+def test_get_all_planets_returns_empty_list_when_db_is_empty(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+    
+def test_all_planets_return_a_list_of_planets(client, two_planets):
+    response = client.get("/planets")
+    response_body = response.get_json()
+    
+    assert response.status_code == 200
+    assert response_body == [{
+        "id": 1,
+        "name": "Jupiter",
+        "description": "Jupiter has sixty-seven moons",
+        "distance_from_sun": "48.4"
+    },
+        {
+        "id": 2,
+        "name": "Mercury",
+        "description": "Mercury has zero moons",
+        "distance_from_sun": "3.5"
+    }]
```

Comment:
Per my comment above in the two_planets fixture, if that fixture returned a list of your 2 saved planets then you could check the response_body against your fixture (where you saved the planets when this test first runs)!

```python
saved_planet_1 = two_planetes[0] 
assert response_body["id"] == saved_planet_1.id 
assert response_body["name"] == saved_planet_1.name
assert response_body["description"] == saved_planet_1.description 
assert response_body["distance_from_sun"] == saved_planet_1.distance_from_sun
```

This would make the test more robust